samples/sample_hevc_fei_abr/include/dso/hevc2_struct.h

```diff
@@ -819,7 +819,7 @@ struct TU
 
     Bs16s QP[3];
 
-    Bs32s* tc_levels_luma = nullptr;
```

Comment:
what was wrong with this one?

Reply:
It made this struct (TU) nontrivial, as a result gcc-8.1 blows up on any memset over it.

Reply:
got it